integration/tests/cook/test_cli.py

```diff
@@ -429,6 +429,10 @@ def list_jobs(self, name, user, *states):
         cp, jobs = cli.jobs_json(self.cook_url, '--name %s --user %s %s' % (name, user, state_flags))
         return cp, jobs
 
+    @unittest.skipIf(util.has_ephemeral_hosts(), 'If the cluster under test has ephemeral hosts, then it is generally '
+                                                 'a bad idea to use HOSTNAME EQUALS constraints, because it can cause '
+                                                 'the process responsible for launching hosts to launch hosts that '
+                                                 'never get used')
     def test_list_by_state(self):
```

Comment:
Is it possible to use a different constraint that causes jobs to not be scheduled and avoid the issue of launching hosts that never get used?

Reply:
The CLI tests don't currently support docker jobs anyway, so I removed the `skipIf` until we address that in a separate PR